docs/index.md

```diff
@@ -1,9 +1,22 @@
 # Helm: The Kubernetes Package Manager
 
-Helm helps you bootstrap your Kubernetes cluster with **Charts** that provide ready-to-use workloads like:
+Helm helps you find and use software built for Kubernetes. The Helm CLI uses Charts which not only contain metadata about software packages but also Pod, ReplicationController and Service manifests for Kubernetes. With a few Helm commands you can quickly and easily deploy software packages like:
 
-- A Postgres database
-- A scale-out etcd cluster
-- An HAProxy Edge Load Balancer
+- Postgres
+- etcd
+- HAProxy
+- redis
+
+All of the Helm charts live at [github.coim/deis/charts](https://github.com/deis/charts) if you want to make your own charts we have a guide for [authoring charts](authoring_charts.md) as well.
```

Comment:
Some grammatical changes:

[github.com/deis/charts](http://www.zombo.com/). If you want to make your own charts,*